consent/models.py

```diff
@@ -239,3 +246,150 @@ class Meta(Consent.Meta):
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>platform_name: Name of platform.</li>'
+        '</ul>'
+    ))
+    SIDEBAR_PARAGRAPH_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for left sidebar paragraph. The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>platform_name: Name of platform.</li>'
+        '</ul>'
+    ))
+    CONFIRMATION_MODAL_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for dialog which appears when user decline to provide consent. '
+        'The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>item: A string which is "course" or "program" depending on the type of consent.</li>'
+        '</ul>'
+    ))
+
+    page_title = models.CharField(
+        max_length=255,
+        default=_('Data sharing consent required'),
+        help_text=_('Title of page')
+    )
+    left_sidebar_header = models.CharField(
+        max_length=255,
+        default=_('Welcome to {platform_name}.'),
+        help_text=_('Header text in left sidebar. platform_name variable may be available')
+    )
+    left_sidebar_text = models.TextField(
+        null=True,
+        blank=True,
+        default=_(
+            "{enterprise_customer_name} has partnered with {platform_name} to offer you high-quality learning "
+            "opportunities from the world's best universities."
+        ),
+        help_text=SIDEBAR_PARAGRAPH_HELP_TEXT
+    )
+    message_header = models.TextField(
+        null=True,
+        blank=True,
+        default=_('Consent to share your data'),
+        help_text=_('Heading of page')
+    )
+    top_paragraph = models.TextField(
+        null=True,
+        blank=True,
+        default=_('First paragraph of page'),
+        help_text=TOP_PARAGRAPH_HELP_TEXT
+    )
+    agreement_text = models.TextField(
+        null=True,
+        blank=True,
+        default=_(
+            'I agree to allow {platform_name} to share data about my enrollment, completion and performance '
+            'in all {platform_name} courses and programs where my enrollment is sponsored by '
+            '{enterprise_customer_name}.'
+        ),
+        help_text=_('Text next to agreement check mark')
+    )
+    continue_text = models.CharField(
+        max_length=255,
+        default=_('Yes, continue'),
+        help_text=_('Text of agree button')
+    )
+    abort_text = models.CharField(
+        max_length=255,
+        default=_('No, take me back.'),
+        help_text=_('Text of decline link')
+    )
+    policy_dropdown_header = models.CharField(
+        max_length=255,
+        null=True,
+        blank=True,
+        default=_('Data Sharing Policy'),
+        help_text=_('Text of policy drop down')
+    )
```

Comment:
We might needy another field like show_policy_details that is a boolean. If true, then we allow displaying policy_dropdown_header and policy_paragraph and if false we do not show those fields. We need this for the Boeing/MIT use case where they do not show the policy dropdown or text at all (see attached screenshot)
![boeingdsc](https://user-images.githubusercontent.com/27503539/37465611-030e1b80-2832-11e8-86c8-50b5c0963a5e.png)

Reply:
I was thinking of displaying policy section only if both `policy_dropdown_header ` and `policy_paragraph` are not empty.